src/adapter/breakpoints/breakpointBase.ts

```diff
@@ -434,7 +440,8 @@ export abstract class Breakpoint {
       bp =>
         (script.url &&
           isSetByUrl(bp.args) &&
-          new RegExp(bp.args.urlRegex ?? '').test(script.url) &&
+          (new RegExp(bp.args.urlRegex ?? '').test(script.url) ||
+            (script.urlRegexp && bp.args.urlRegex === script.urlRegexp)) &&
```

Comment:
Since we never set by URL, this function signature and logic can be simplified

Reply:
Deleted